src/views/Simulation/Simulation.js

```diff
@@ -20,8 +25,9 @@ const Simulation = () => {
           </Grid>
           <Grid item xs={12}>
             <div id="drawer-container" style={{ position: 'relative', height: '100%' }}>
-              <Scene />
+              {viewMode === SIMULATION_MODES.GRAPH ? <Scene /> : <MapView />}
               <InspectorDrawer containerRef={inspectorDrawerParentContainerRef} />
+              <ModeSwitcher mode={viewMode} onChange={setViewMode} />
```

Comment:
The `ModeSwitcher` is hidden behind the timeline component when the animation is enabled. Can we hide the `ModeSwitcher` when the timeline is visible?